docs/migrating-configs.md

```diff
@@ -110,32 +110,6 @@ The `luks` section gained a new `openOptions` field. It is a list of options Ign
 }
 ```
 
-### Special mode bits supported
```

Comment:
Hum, I think we should keep that but update the text to mention the partial fix and the versions?

Reply:
Hmm, it did not work though; so we would be documenting knowingly broken feature. I guess we could say this feature has a breaking bug, and will not function as expected?

Reply:
Ah indeed, this is for the older spec. We should then move it at the top of the doc for the upcoming spec.